pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -121,10 +122,12 @@ public void calculateBrokerHostUsage() {
         this.usage = usage;
     }
 
-    private double getTotalNicLimitWithConfiguration(List<String> nics) {
+    @VisibleForTesting
+    public double getTotalNicLimitWithConfiguration(List<String> nics) {
```

Comment:
it's a public method, so can delete `@VisibleForTesting`

Reply:
`@VisibleForTesting ` is used to notify this method public for testing. If I change it to private the test can not work anymore.

Reply:
change to protected